main/realmain.cc

```diff
@@ -301,17 +301,22 @@ void runAutogen(const core::GlobalState &gs, options::Options &opts, const autog
                 continue;
             }
 
-            for (const auto &[parentName, children] : *el.subclasses) {
-                if (!parentName.empty()) {
-                    auto &childEntry = childMap[parentName];
+            for (const auto &[parentRef, children] : *el.subclasses) {
+                if (parentRef.exists()) {
```

Comment:
Do you think we could also remove this .exists() call? I'm not sure what the `.empty()` check was there for before, so I'm not sure if this is caused by something else.